TTS/speaker_encoder/models/resnet.py

```diff
@@ -88,6 +107,21 @@ def __init__(
 
         self.instancenorm = nn.InstanceNorm1d(input_dim)
 
+        if self.use_torch_spec:
+            self.torch_spec = torch.nn.Sequential(
+                PreEmphasis(audio_config["preemphasis"]),
+                torchaudio.transforms.MelSpectrogram(
```

Comment:
I tried to do the TorchSTFT compatible with the torchaudio.transforms.MelSpectrogram but have a lot of things differently.  I found and fix a lot of differences but It wasn't enough I think it will take a long time to turn this compatible.  Right now I don't have enough time for this :(. As we are using the checkpoint provided by the authors in the official implementation we need to keep the torchaudio.

Reply:
I removed torchaudio as a requirement. But now it is no longer possible to use the speaker consistency loss when the sample rate of the speaker encoder and the TTS model are different. I left the code that uses torchaudio for resample commented out. This way, if you find an efficient way to resample an audio batch without using torchaudio you can easily implement that.